build.sbt

```diff
@@ -2,6 +2,7 @@ ThisBuild / scalaVersion     := "2.12.10"
 ThisBuild / version          := git.gitHeadCommit.value.getOrElse("1.0")
 ThisBuild / organization     := "com.commercetools"
 ThisBuild / organizationName := "commercetools"
+Test     / parallelExecution := false
```

Comment:
You also have to disable the parallel execution in the runner: https://github.com/commercetools/ctp-smoke-tests/blob/master/build.sbt#L35

Maybe we can disable the parallel execution in another PR? That would be easier to revert at a later point.

Reply:
done, created new PR